src/libs/antares/writer/in_memory_writer.cpp

```diff
@@ -77,27 +84,30 @@ void InMemoryWriter::addEntryFromBuffer(const std::string& entryPath, std::strin
     addToMap(pEntries, entryPath, entryContent, pMapMutex, pDurationCollector);
 }
 
-void InMemoryWriter::addEntryFromFile(const std::string& entryPath, const std::string& filePath)
+static std::string readFile(const fs::path& filePath)
```

Comment:
For this kind of very generic function (reading a file to a string), we could share it in a technical lib, for example we already have an "IO" library where it could go.

This function would throw an exception on failure, which we could just let through here.